### DISCUSSION TITLE
Data importer: Populating Many-to-Many Object relation fields

### DISCUSSION THREAD

**Discussion by sissonen** [pimcore/pimcore#9832] [Q&A / Support] [answered]
Data importer has the "Load DataObject" operator, but it cannot be used to populate Many-to-Many Object relation fields (the transformation result type is "dataObject" -> Data target with "Direct" does not show the fields which are of type "dataObjectArray"). I tried using "As Array" operator in the pipe after loading the data object that but that only resulted in "array" type.

Is there already a way to do this?

I could try implementing this but should the implementation be: 1) new operator "Load DataObject Array" or 2) a checkbox in "Load DataObject" (e.g. "Transform result into DataObject Array")?

**Comment by fashxp** [comment, marked as answer]
what if you make sure that the input for the Load DataObject operator is already an array? So put the As Array operator in front?

**Reply by sissonen** [reply]
Yes, that did the trick. Thanks!
